spoonacular/history.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": 1,
+    "request": {
+        "url": "https://api.spoonacular.com/recipes/findByNutrients?",
+        "method": "GET",
+        "authorization": {
+            "type": "TokenParam",
+            "keyParam": "apiKey"
+        }
+        "params": {
+            "minProtein": false,
```

Comment:
at least one of these parameters is required

Reply:
Thanks for pointing out! We currently do not have the syntax for "at least one is required". Maybe we can do that in the future.